models/articles.model.js

```diff
@@ -4,8 +4,26 @@ const { doesTopicExist } = require("./topic.model");
 exports.selectArticle = (id) => {
   return db
     .query(
-      `SELECT * FROM articles
-  WHERE article_id=$1;`,
+      `SELECT
+      articles.author,       articles.title,
+      articles.article_id,       articles.body,
+      articles.topic,
+      articles.created_at,
+      article_img_url,
+      articles.votes,
+
+      COUNT(comments.comment_id)::INT AS comment_count
+      FROM articles
+      LEFT OUTER JOIN comments
+      ON articles.article_id=comments.article_id
+      WHERE articles.article_id=$1
+      GROUP BY articles.article_id,
+      articles.author,       articles.title,
+      articles.article_id,       articles.body,
+      articles.topic,
+      articles.created_at,
+      article_img_url,
+      articles.votes;`, 
```

Comment:
My esteemed colleague Pippa has come across this before, we shouldn't need to group by all these however, just article.article_id! 🚀 It should work then